az_quickstart.info.yml

```diff
@@ -41,7 +43,6 @@ install:
   - menu_ui
   - options
   - path
-  - pathauto
```

Comment:
az_seo enables this, otherwise, it isn't needed

Reply:
Should be included in the quickstart install profile.